### PR TITLE
Add page aa1 to master

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Documentation (non-breaking change which adds documentation)
 - [x] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently the entire page is just a massive template.

**What is the new behavior (if this is a feature change)?**
- The template has been boiled down and a beautiful website has been created.

**Other information**:
- The version stays `v1.0.0` once this will be the initial release.

### CODE REVIEW

File: src/components/pages/HomePage/index.jsx [hunk 1]
```diff
@@ -30,83 +35,159 @@ class HomePage extends React.Component {
...
+                        We are looking for collaborators
+                      </strong>
+                    </h2>
+                    <MDBRow />
+                    <p className="lead mb-1">
+                      We <MDBIcon icon="heart" className="pink-text" /> Open
+                      Source
+                    </p>
+                    <p className="pb-2 text-muted">
+                      All of our tool, research and progress is{" "}
```
Comment (Theralun) [inline src/components/pages/HomePage/index.jsx]: tools?

**Review by Theralun — COMMENTED** [review]
Have a look at this.

**Review by Theralun — APPROVED** [review]
lgtm.

**Review by kleberbaum — APPROVED** [review]
Should be ok to merge.

**Pull request merged**